src/bvals/comms/bnd_info.cpp

```diff
@@ -303,17 +306,6 @@ BndInfo BndInfo::GetSetBndInfo(std::shared_ptr<MeshBlock> pmb, const NeighborBlo
     }
   }
 
-  if (buf_state == BufferState::received) {
-    // With control variables, we can end up in a state where a
-    // variable that is not receiving null data is unallocated.
-    // for allocated to be set, the buffer must be sending non-null
-    // data and the receiving variable must be allocated
```

Comment:
Is this comment still relevant?

Reply:
It is still relevant but I think this is not the right place to have it.

Reply:
just noticed it's not added anywhere else. If relevant maybe stick it somewhere else in the source?

Reply:
Thinking about this a little more, this is naturally taken care of now that I disambiguated the `BndInfo::allocated`. The comment probably doesn't need to be anywhere.

Reply:
cool :+1: